src/package_io/parse_input.star

```diff
@@ -16,6 +16,7 @@ DEFAULT_CL_IMAGES = {
 }
 
```

Comment:
should we change the count of the default participant to 2? we change network-params.json; if we do lets close #200 by this ticket

Reply:
We can't do that. As if I do that, then all defaults will change, so you will manually have to set that you want a count of 1, if you want a count of 1.